### PR TITLE
Stop Disk from relying on nested params and make Disk KMS fields GA

### PR DESCRIPTION
**Pull request by chrisst**
Move the encryption properties out of the shared erb and in line with Disk
This will also make KMS fields "GA"

-----------------------------------------------------------------
# [all]
Make Disk KMS features GA

### CODE REVIEW

File: products/compute/api.yaml [hunk 1]
```diff
@@ -659,7 +658,85 @@ objects:
           that was later deleted and recreated under the same name, the source
           image ID would identify the exact version of the image that was used.
         output: true
-<%= indent(compile('products/compute/disk_parameters.yaml'), 6) %>
```
Comment (rileykarson) [inline products/compute/api.yaml]: Want to inline this into `RegionDisk` too and delete it?

File: products/compute/api.yaml [hunk 2]
```diff
@@ -659,7 +658,85 @@ objects:
...
+      - !ruby/object:Api::Type::NestedObject
```
Comment (rileykarson) [inline products/compute/api.yaml]: Have any lines below here been modified? Or is it all copied directly from the other file? My expectation is that a few `min_version: beta` lines have been removed based on the downstreams, but as-is this diff doesn't make that clear.

Ideally if this is a copy-and-change the copy and change would be broken into separate commits.

File: products/compute/api.yaml (hunk 1 above)
Comment (chrisst) [reply products/compute/api.yaml]: I'm happy to do that in a separate PR. Pulling it into this PR is scope creep that's not really required.

File: products/compute/api.yaml (hunk 2 above)
Comment (chrisst) [reply products/compute/api.yaml]: Ideally, you are correct. But the commit message indicates that both happened, as does the PR title. IMO it's not a change that blocks this PR.

File: products/compute/api.yaml (hunk 1 above)
Comment (rileykarson) [reply products/compute/api.yaml]: I see where you're coming from, but this PR is a 3 line change otherwise; I think it's a perfectly reasonable PR to inline the file in both places and then making this minor change as a separate commit. You're also correct that splitting it into two distinct PRs would be even better imo.

Comment (chrisst) [reply products/compute/api.yaml]: I'll follow up with the rest of the refactors in another PR 👍
